sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/api/Azure.AI.Language.QuestionAnswering.netstandard2.0.cs

```diff
@@ -181,20 +189,12 @@ public partial class TextAnswers
         internal TextAnswers() { }
         public System.Collections.Generic.IReadOnlyList<Azure.AI.Language.QuestionAnswering.Models.TextAnswer> Answers { get { throw null; } }
     }
-    public partial class TextInput
+    public partial class TextRecord
```

Comment:
not related to the PR but curious about the reason for the name change. `TextInput` was closer to TA and in case we want some sort of consistency in CS, maybe having `TextInput` here is not a bad idea either

Reply:
I thought you used `TextRecord` - the idea to change it came from TA, or maybe I saw that before it changed. That said, I do know there was an effort to refer to these as records across the pillar.

Reply:
I opened https://github.com/Azure/azure-rest-api-specs/issues/15374 to track for next release.